app/views/spree/layouts/_admin_body.html.haml

```diff
@@ -10,11 +10,29 @@
     .flash.success= flash[:success]
 
   #progress
-    .wrapper
-      #spinner
-      .progress-message
-        = Spree.t(:loading)
-        \...
+    / By Sam Herbert (@sherb), for everyone. More @ http://goo.gl/7AJzbL
```

Comment:
you are inlining an SVG directly in a layout view? It doesn't look good. Any alternative?

Reply:
we'll need to move it to a partial and reuse it in app/views/spree/layouts/bare_admin.html.haml too. Would that feel ok @luisramos0 ?

Reply:
yes, a specific partial is better, but why not a svg file?

Reply:
because it is then impossible to style it with CSS and style the spinner different than the image we have now. I understand you might be skeptical but it's an incredibly common technique to replace icon fonts, for instance. Just check your GH's dashboard and look up `svg`. It's basically a lot more changeable.

Reply:
hmmm, interesting, I didnt know that, thanks!
So let's move it to a partial?
Shall we have a specific folder for svgs in app/views?

Reply:
> Shall we have a specific folder for svgs in app/views?

I think it's too early for that. We don't even know if we'll do this again any time soon. It's something we can consider if it happens.

Reply:
ok :+1: